### PR TITLE
Redeem Brave Ads confirmation tokens after 3 seconds

### PR DESCRIPTION
**Pull request by tmancey**
https://github.com/brave/brave-variations/issues/1264

### CODE REVIEW

**Comment by btlechowski** [comment]
Verified on

Brave | 1.73.91 Chromium: 131.0.6778.85 (Official Build) (64-bit)
-- | --
Revision | 35427fa72a008a7e861fa9828e07a0ba2cd46ebd
OS | Linux


Verified the study is enabled
image

Verified the fetch is 3s
```
[9279:9279:1120/170720.378798:VERBOSE1:redeem_reward_confirmation.cc(123)] Fetch payment token in 3 s
```

--------

Verification passed on Brave v1.73.91 on Lenovo TB-8506FS (Android 11.0)

Verified the study is enabled

image


Verified the fetch is 3s
```
18:15:15.188  V  [VERBOSE1:redeem_reward_confirmation.cc(123)] Fetch payment token in 3 s
```

**Review by kjozwiak — APPROVED** [review]
Uplift into `main/production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR as per https://github.com/brave/brave-variations/pull/1265#issuecomment-2489151997.

**Pull request merged**
